DependencyInjection/Configuration.php

```diff
@@ -107,7 +107,11 @@ public function getConfigTreeBuilder()
                                 return $v;
                             })
                         ->end()
-                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))->end()
+                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))
+                    ->end()
+                    ->scalarNode('service_naming_strategy')
```

Comment:
I needed it for an internally reusable bundle but since my use case isn't the most common I guess I can handle using a compiler pass override and everyone else can use application-level configs.